src/map/lua/lua_baseentity.h

```diff
@@ -233,6 +233,7 @@ class CLuaBaseEntity
     uint8 getFreeSlotsCount(sol::object const& locID);         // Gets value of free slots in Entity inventory
     void  confirmTrade();                                      // Complete trade with an npc, only removing confirmed items
     void  tradeComplete();                                     // Complete trade with an npc
+    void  tradeRelease();                                      // Complete trade with an npc without taking any item, but properly "unreserving" them
```

Comment:
is this a counterpart to confirm or complete? This makes it look like its an opposite to complete.

Reply:
This IS an oposite to complete

Reply:
I had to confirm we didn't mean confirm :D 

so many trade thingies, and I thought there was some problem with items being added and then no confirm call to eat them.